src/app/services/appointments.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
```

Comment:
this will load many component. you can do 
import { Observable } from 'rxjs/Observable';
import 'rxjs/add/operator/map';